src/applications/facility-locator/actions/index.js

```diff
@@ -404,3 +407,20 @@ export const getProviderSpecialties = () => async dispatch => {
     return ['Services Temporarily Unavailable'];
   }
 };
+
+export const geolocateUser = () => async dispatch => {
+  if (navigator.geolocation) {
+    dispatch({ type: GEOCODE_STARTED });
+    navigator.geolocation.getCurrentPosition(async currentPosition => {
+      const query = await searchCriteraFromCoords(
+        currentPosition.coords.longitude,
+        currentPosition.coords.latitude,
+      );
+      dispatch({ type: GEOCODE_COMPLETE });
+      dispatch(updateSearchQuery(query));
+    });
+  } else {
+    // eslint-disable-next-line no-console
+    console.warn('navigator.geolocation undefined; unable to geolocate user');
```

Comment:
This warning is for cases where navigator.geolocation is undefined. But that's probably extremely rare since it's been supported for a long time in every browser:

https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition

We could also log an error when geolocation fails though. I'll go ahead and add that.